casper/src/main/scala/coop/rchain/casper/genesis/contracts/ProofOfStake.scala

```diff
@@ -24,6 +22,7 @@ final case class ProofOfStake(minimumBond: Long, maximumBond: Long, validators:
 }
 
 object ProofOfStake {
+  val (_, pk) = Secp256k1.newKeyPair
```

Comment:
This is going to differ across different node runs. We must hardcode it.

Reply:
We should use the PK corresponding to the genesis deploy, or make that one refer to this one

Reply:
There are at least two places (`Genesis.scala` and `BlockApproverProtocol.scala`) where we generate the genesis pk in this way.
Therefore I'm not sure if hardcoding it just here or passing it from the genesis deploy would help much.
How about adding a comment and creating a ticket where we solve the problem in its entirety?

Reply:
I think this pk is currently unused (we don't use deployerId in PoS.rhox). And we shouldn't be, really. The PoS vault should be an unf-name based vault, with a multisig wrapper over it - tho PoS would use the unf-name based one though. Please make the corresponding RholangSource/Template use EmptyEnv

Reply:
Done